platform/experimental/BUILD

```diff
@@ -0,0 +1,49 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
rename directory 'platforms'

Reply:
set license headers

Reply:
done.
Also rename toolchain to toolchains if that's fine with you